pom.xml

```diff
@@ -121,6 +125,11 @@
         <artifactId>juel-impl</artifactId>
         <version>2.2.7</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.re2j</groupId>
+        <artifactId>re2j</artifactId>
+        <version>1.2</version>
```

Comment:
duplicated dependency?

Reply:
no, this is specifying the version.

Reply:
So, is this the correct way to include a dep and specify a version? If there is a better way, I can adjust

Reply:
You can just put the version in with the dependency. I'm not sure why some are separated out in this pom.